styles/base.less

```diff
@@ -1,4 +1,649 @@
 
+/* IT'S DA BOY KODY AT IT AGAIN */
+
+.field-list  {
+  font-family: proxima-nova;
+}
+
+.lp-row {
+  width: 100%;
+  float: left;
+}
+
+.lp-signup-top {
+  position: fixed;
+  top: 8.5%;
+  right: 10%;
+  z-index: 999;
+}
+
+.lp-signup-open {
+  cursor: pointer;
+}
+
+.lp-signup-top span {
+  border: solid 3px;
+  padding: 6px 18px;
+  cursor: pointer;
+}
+
+#header #logoWrapper {
+  display: none!important;
+}
+
+.lp-logo {
+  position: fixed;
+  left: 5%;
+  top: 5%;
+  z-index: 999;
+}
+
+.lp-logo img {
+  width: 100px;
+  height: 100px;
+}
+
+.form {
+  text-align: right;
+}
+
+.form-tradie-active {
+  height: auto!important;
+  opacity: 1!important;
+}
+
+.form-tradie-2, .form-tradie-1 {
+  height: 0;
+  opacity: 0;
+}
+
+.field-wrap {
+  float: left;
+  width: 48%;
+  margin: 1%;
+}
+
+.field-wrap-message {
+  float: left;
+  width: 98%;
+  margin: 1%;
+}
+
+input, textarea, select {
```

Comment:
so how does this work with other pages?

e.g. see `about` and `registration`:

<img width="1111" alt="screen shot 2018-07-10 at 3 20 39 pm" src="https://user-images.githubusercontent.com/629710/42490737-2f6d09c8-8455-11e8-9aa7-708b28723cef.png">
<img width="1289" alt="screen shot 2018-07-10 at 3 20 54 pm" src="https://user-images.githubusercontent.com/629710/42490740-313c9a98-8455-11e8-9f2b-421e49ad811f.png">

Reply:
I've amended this to be targeting the input, textarea and select inside .signup-form only